.gitlab-ci.yml

```
@@ -41,7 +42,8 @@ test:acceptance_tests:
     - hetzner-amd-beefy
   image: tiangolo/docker-with-compose
```

```suggestion
  image: ${CI_DEPENDENCY_PROXY_DIRECT_GROUP_IMAGE_PREFIX}/tiangolo/docker-with-compose
```

This to avoid docker hub rate limit issue